### ISSUE TITLE
Widget shows all calendars including hidden ones

### ISSUE THREAD

**Issue by dufayf** [adolfintel/Clock31#6]
Hi,

the widget is very nice and clean. But it will show every calendars of every accounts of the phone, creating confusion and duplicating events in case of shared calendars between accounts.

Could it be possible to filter the agendas by interrogating the agenda application (or Google's agenda application in most case) on which calendar is shown or hidden? It should reflect the real use of our own agenda application on our phone.

Thanks

**Comment by adolfintel** [comment]
Yeah I know, I haven't implemented settings for the widget yet so they just show all calendars which works for me.
I'll probably do it at some point, but I'm currently trying to use a linux phone and I have a full time job keeping me busy so it may take a while.
